package.json

```diff
@@ -21,18 +20,18 @@
   "scripts": {
     "lint": "tslint -p ./tsconfig.json",
     "start": "react-scripts start",
-    "clean": "rm -rf build",
-    "build": "GENERATE_SOURCEMAP=true react-scripts build",
+    "clean": "rm -rf ./build",
+    "build": "GENERATE_SOURCEMAP=${GENERATE_SOURCEMAP:-true} react-scripts build",
     "build:clean": "npm run-script clean && npm run-script build",
     "prerender": "node ./script/entry prerender",
-    "server": "NODE_ENV=development node script/entry server/cli",
+    "server": "NODE_ENV=${NODE_ENV:-development} node ./script/entry server/cli",
     "codecov": "codecov",
     "postinstall": "node ./script/postinstall",
     "test": "npm run-script test:unit && npm run-script lint && npm run-script test:build && tsc",
     "test:unit": "jest --coverage --testPathPattern=\"(\\.|/)(browser)?spec\\.tsx?\"",
     "test:build": "npm run-script build:clean && npm run-script test:sourcemap && npm run-script test:prerender",
     "test:sourcemap": "node ./script/verifySourcemaps.js",
-    "test:prerender": "NODE_ENV=test npm run-script prerender && SERVER_MODE=built jest --testPathPattern=\"(\\.|/)(prerender|browser)spec\\.tsx?\""
+    "test:prerender": "NODE_ENV=${NODE_ENV:-test} npm run-script prerender && SERVER_MODE=built jest --testPathPattern=\"(\\.|/)(prerender|browser)spec\\.tsx?\""
```

Comment:
`test:prerender` is never going to work outside `NODE_ENV=test`, we shouldn't allow changing it here